### ISSUE TITLE
botocore.vendored.requests deprecation

### ISSUE THREAD

**Issue by jantman** [jantman/awslimitchecker#421]
Running awslimitchecker with the latest versions of all dependencies, we get the following DeprecationWarning:

```
lib/python2.7/site-packages/botocore/vendored/requests/api.py:67: DeprecationWarning: You are using the get() function from 'botocore.vendored.requests'.  This is not a public API in botocore and will be removed in the future. Additionally, this version of requests is out of date.  We recommend you install the requests package, 'import requests' directly, and use the requests.get() function instead.
```

This was recently changed to display by default in https://github.com/boto/botocore/pull/1748 and was previously added in https://github.com/boto/botocore/pull/1625 and overall relates to https://github.com/boto/botocore/issues/1608 which calls for the removal of botocore's vendored-in ``requests`` and ``urllib3``, as they're no longer used. This was announced in [the 1.11.0 botocore CHANGELOG entry](https://github.com/boto/botocore/blob/develop/CHANGELOG.rst#1110), which was [released](https://github.com/boto/botocore/releases/tag/1.11.0) back on August 24, 2018.

It appears that we're currently using the vendored requests in three places:

* [services/efs.py](https://github.com/jantman/awslimitchecker/blob/8312a100012c456f13700ab1b45449e494695840/awslimitchecker/services/efs.py#L43) and [tests/services/efs.py](https://github.com/jantman/awslimitchecker/blob/8312a100012c456f13700ab1b45449e494695840/awslimitchecker/tests/services/test_efs.py#L44) both ``from botocore.vendored.requests.exceptions import ConnectTimeout``, which was added in #340 to fix #335 
* ``utils.py`` uses the ``get()`` method of the vendored requests package for its [_get_latest_version() method](https://github.com/jantman/awslimitchecker/blob/8312a100012c456f13700ab1b45449e494695840/awslimitchecker/utils.py#L216)

The first one appears that, since the release of https://github.com/boto/botocore/pull/1495 ("Unvendor requests and use urllib3 directly") a year ago, we should be able to catch either ``botocore.exceptions.ConnectTimeoutError`` or its other parent class, ``botocore.exceptions.ConnectionError``.

For the second one, checking the latest awslimitchecker version, botocore itself now depends on the normal upstream urllib3 package. We should too, and should just use that.

**Comment by rohanjoseff** [comment]
Does the docker image tagged "master" have this fix?

**Comment by jantman** [comment]
@rohanjoseff It does not. This fix is currently staged in the ``develop`` branch, waiting on a few more feature requests before I cut a release. This should be done in a week or so.

Please keep in mind that the vendored-in requests library will still be present in the installation until https://github.com/boto/botocore/issues/1608 is done upstream; awslimitchecker will just not be using it anymore.

**Comment by jantman** [comment]
This was merged to master in #427 and released as [7.1.0](https://github.com/jantman/awslimitchecker/releases/tag/7.1.0). The release is now [live on PyPI](https://pypi.org/project/awslimitchecker/) and the Docker image should be live [on Docker Hub](https://hub.docker.com/r/jantman/awslimitchecker) shortly.

@rohanjoseff
